### PR TITLE
Client sends docket results and test refactor

### PR DESCRIPTION
**Pull request by MrTsfree**

### CODE REVIEW

**Comment by kylienorwood** [comment]
We should be getting up the pylint corrections shortly.

In terms of the code, let Trae and I know what you think about the tests. We made a very simple results JSON example for mocking, but weren't sure if we should use one that looks more exact to the docket results. 

This also includes the changes to the put_results endpoint format, which I sent a message about in the Discord. Any feedback is appreciated!!

File: tests/c21client/test_client.py [hunk 1]
```diff
@@ -84,7 +86,7 @@ def test_client_sends_job_results(mock_requests, mocker):
         )
         mock_requests.put(
             f'{BASE_URL}/put_results',
-            json={'success': 'The job was successfully completed'},
```
Comment (rileyalankirk) [inline tests/c21client/test_client.py]: This change doesn't make sense. This line is mocking out what the put_results endpoint returns which was correct before. The change makes the endpoint return the data that is supposed to be sent to it.

Comment (kylienorwood) [reply tests/c21client/test_client.py]: He is right... @MrTsfree just replace `json=mock_job_result` with `json={'success': 'The job was successfully completed'},` like it was before.

Comment (MrTsfree) [reply tests/c21client/test_client.py]: I went ahead and rolled this back to `json={'success': 'The job was successfully completed'}`.  Sorry about that!  Check my latest commit for the change.

File: src/c21client/client.py [hunk 2]
```diff
@@ -34,7 +34,12 @@ def get_job(self):
     def send_job_results(self, job_id, job_result):
         endpoint = f'{self.url}/put_results'
         client_id = self.client_id
-        data = {'results': {job_id: job_result}, 'client_id': client_id}
+        agency_id = job_result['data']['agencyId']
+        docket_id = job_result['data']['id']
+        data = {'client_id': client_id,
+                'directory': f'{agency_id}/{docket_id}',
```
Comment (rileyalankirk) [inline src/c21client/client.py]: Should include the file name: `f'{agency_id}/{docket_id}'/{docket_id}.json`

Comment (MrTsfree) [reply src/c21client/client.py]: Went ahead and fixed this.  Check latest commit for changes!

**Pull request merged**